sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -237,6 +246,99 @@ def get_custom_model(self, model_id, **kwargs):
+        """
+
+        response = self._client.generate_model_copy_authorization(  # type: ignore
+            cls=lambda pipeline_response, deserialized, response_headers: pipeline_response,
+            error_map=error_map,
+            **kwargs
+        )  # type: PipelineResponse
+        target = json.loads(response.http_response.text())
+        target["resourceId"] = resource_id
+        target["resourceRegion"] = resource_region
+        return target
+
+    @distributed_trace
+    def begin_copy_model(
+        self,
+        model_id,  # type: str
+        target,  # type: Dict
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> LROPoller
+        """Copy a custom model stored in this resource (the source) to the user specified
+        target Form Recognizer resource. This should be called with the source Form Recognizer resource
+        (with the model that is intended to be copied). The `target` parameter should be supplied from the
+        target resource's output from calling the :func:`~authorize_copy_target()` method.
+
+        :param str model_id: Model identifier of the model to copy to target resource.
+        :param dict target:
+            The copy authorization generated from the target resource's call to
+            :func:`~authorize_copy_target()`.
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if
+            no Retry-After header is present.
+        :return: An instance of an LROPoller. Call `result()` on the poller
+            object to return a :class:`~azure.ai.formrecognizer.CustomFormModelInfo`.
+        :rtype: ~azure.core.polling.LROPoller[CustomFormModelInfo]
+        :raises ~azure.core.exceptions.HttpResponseError:
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_copy_model.py
+                :start-after: [START begin_copy_model]
+                :end-before: [END begin_copy_model]
+                :language: python
+                :dedent: 8
+                :caption: Copy a model from the source resource to the target resource
+        """
+
+        polling_interval = kwargs.pop("polling_interval", POLLING_INTERVAL)
+
+        def _copy_callback(raw_response, _, headers):  # pylint: disable=unused-argument
+            copy_result = self._client._deserialize(CopyOperationResult, raw_response)
+            return CustomFormModelInfo._from_generated_copy(copy_result, target["modelId"])
+
+        return self._client.begin_copy_custom_model(  # type: ignore
+            model_id=model_id,
+            copy_request=CopyRequest(
+                target_resource_id=target["resourceId"],
+                target_resource_region=target["resourceRegion"],
+                copy_authorization=CopyAuthorizationResult(
+                    access_token=target["accessToken"],
+                    model_id=target["modelId"],
+                    expiration_date_time_ticks=target["expirationDateTimeTicks"]
```

Comment:
Is it necessary to call this `expiration_date_time_ticks`? I'm not seeing other expiration dates, so could this be called `expiration_date`?

p.s. I don't feel very strongly about this, just wanted to raise the question

Reply:
talked offline